README.md

```diff
@@ -412,6 +413,12 @@ data conversion can be found here:
 - [CSV data conversion](sources/csv/README.md#example-csv-usage)
 - [SQL Server data conversion](sources/sqlserver/README.md#data-conversion)
 
+### Data Migration Recommendations
+- We recommend to use this data migration solution only for small databases 
+(smaller than 100GB) without strict downtime requirements. 
+- While using direct connect, it is recommended to use a secondary/read replica
+ to ensure consistency and that there is no impact from the load.
```

Comment:
Nit: wordy
Suggest rephrasing too "and ~~that there is~~ no impact from the load"

Reply:
How about "to ensure consistency and avoid impact from the load on primary replica"